R/repr_matrix_df.r

```diff
@@ -161,7 +173,10 @@ repr_latex.matrix <- function(obj, ..., colspec = getOption('repr.matrix.latex.c
 	obj <- latex.escape.names(obj)
 	
 	# escape LaTeX special characters if necessary
-	if (any(apply(obj, 2L, any.latex.specials))) {
+	need_to_excape_col <- vapply(seq_len(ncol(obj)), function(col_index) {
+			any.latex.specials(obj[, col_index])
+		}, FUN.VALUE = TRUE)
```

Comment:
we just used regular apply here, not dplyr.

but maybe we should just document this (so nobody tries to “fix”) it and move on.